### PR TITLE
Cloudwatch

### PR DESCRIPTION
**Pull request by arunbhagyanath**
### Description

Cloudwatch implementation which supports create/update, disable_action, enable_action and delete cloudwatch alarm metrics.

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-end
-
-chef_version '>= 12.1'
+name 'aws'
+maintainer 'Chef Software, Inc.'
+maintainer_email 'cookbooks@chef.io'
+license 'Apache 2.0'
+description 'Custom resources for managing AWS resources'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+version '4.1.4'
```
Comment (tas50) [inline metadata.rb]: Can you revert this version bump. We'll probably release this as 4.2.0, but we have tooling that handles the changelog and bump automatically.

Comment (arunbhagyanath) [inline metadata.rb]: Reverted in commit #88c8507

File: metadata.rb [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+version '4.1.3'
+
+recipe 'aws', 'Installs the aws-sdk gem during compile time'
+recipe 'ec2_hints', 'Adds an EC2 hint file for Ohai cloud detection'
+
+source_url 'https://github.com/chef-cookbooks/aws'
+issues_url 'https://github.com/chef-cookbooks/aws/issues'
+depends 'ohai', '>= 4.0'
+
+%w(ubuntu debian centos redhat amazon scientific fedora oracle freebsd windows suse opensuse opensuseleap).each do |os|
```
Comment (tas50) [inline metadata.rb]: Any idea what's changing every line in some of these files?

**Comment by tas50** [comment]
This is a super useful new addition. Can you please open a PR to document it though so that everyone knows this is an option?

File: metadata.rb (hunk 2 above)
Comment (arunbhagyanath) [inline metadata.rb]: No idea.

**Comment by arunbhagyanath** [comment]
Oops, missed to commit the README.md. Create a new PR #268

**Pull request merged**